spyder/utils/code_analysis/__init__.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+
+"""Code introspection and linting utillites."""
+
+from spyder.config.base import DEV
+
+
+# Language server communication verbosity at server logs.
+TRACE = 'messages'
+if DEV:
+    TRACE = 'verbose'
+
+
+# Spyder editor capabilities
+EDITOR_CAPABILITES = {
+    "workspace": {
```

Comment:
I don't know what these terms refer to. Could you add a comment above each one to understand what they mean?